odbp.client/src/components/UtrechtTableContainer.vue

```diff
@@ -6,7 +6,7 @@
 
 <style lang="scss" scoped>
 .gpp-woo-table__container {
-  overflow-inline: auto;
+  overflow-x: auto;
```

Comment:
is dit een specificity-dingetje? Of moet het idd de x-as zijn, ongeacht de tekstrichting?